code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -874,6 +874,29 @@
 	name = "warm [name]"
 	processing_objects.Add(src)
 
+/obj/item/weapon/reagent_containers/food/snacks/donkpocket/self_heating/
+	name = "self-heating Donk-pocket"
+	icon_state = "donkpocket_wrapped"
+	desc = "Individually wrapped, frozen, unfrozen, desiccated, ressicated, twice recalled, and still edible. Infamously so."
+	wrapped = TRUE
+
+/obj/item/weapon/reagent_containers/food/snacks/donkpocket/self_heating/attack_self(mob/user)
+	if(wrapped)
+		Unwrap(user)
+	else
+		..()
+
+/obj/item/weapon/reagent_containers/food/snacks/donkpocket/self_heating/proc/Unwrap(mob/user)
+	playsound(src, 'sound/misc/donkselfheat.ogg', 35, 0, -4)
+	to_chat(user, "<span class='notice'>Following the instructions, you shake the packaging firmly and rip it open with an unsatisfying wet crunch.</span>")
+	spawn(2 SECONDS)
+		name = "\improper Donk-pocket"
+		desc = "Freshly warmed and probably not toxic."
+		icon_state = "donkpocket"
+		reagents.add_reagent(CALCIUMOXIDE, 0.2)
```

Comment:
Calcium oxide doesn't actually heat anything if there is no water, and donk pockets contain none. This is just poison.

Reply:
Yep, it's intended to thematically be a slightly toxic leftover from the self-heating
It does like 1 toxin that instantly gets countered from the warm donkordrazine anyways